### PR TITLE
fix(types): adding correct unsubscribed component props

### PR DESCRIPTION
**Pull request by mateoguzmana**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

 Removing unused props on the `ShowUnsubscribed` component to fix some ts-ignores.

### CODE REVIEW

File: client/src/pages/unsubscribed.tsx [hunk 1]
```diff
@@ -8,12 +7,9 @@ import RedirectHome from '../components/redirect-home';
 function Unsubscribed(): JSX.Element {
   return (
     <Router>
-      {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-      {/* @ts-ignore */}
-      <ShowUnsubscribed path={withPrefix('/unsubscribed/:unsubscribeId')} />
-      {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-      {/* @ts-ignore */}
-      <ShowUnsubscribed path={withPrefix('/unsubscribed')} />
```
[the remark this answers is not in the document]
Comment (mateoguzmana) [reply client/src/pages/unsubscribed.tsx]: Thanks @huyenltnguyen! I've put them back with their correct typings as well.

**Review by ojeytonwilliams — APPROVED** [review]
The updated types look right to me. Thanks!

File: client/src/client-only-routes/show-unsubscribed.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Grid, Panel, Button } from '@freecodecamp/react-bootstrap';
+import { RouteComponentProps } from '@reach/router';
```
Comment (Sboonny) [inline client/src/client-only-routes/show-unsubscribed.tsx:2]: a nitpick, you can use the keyword type to indicate that it's a type for readability.

```suggestion
import type { RouteComponentProps } from '@reach/router';
```

**Review by Sboonny — APPROVED** [review]
LGTM

**Pull request merged**